server/presenters/team.ts

```diff
@@ -16,6 +16,6 @@ export default function present(team: Team) {
     url: team.url,
     defaultUserRole: team.defaultUserRole,
     inviteRequired: team.inviteRequired,
-    allowedDomains: team.allowedDomains.map((d) => d.name),
+    allowedDomains: team.allowedDomains?.map((d) => d.name),
```

Comment:
I am surprised this wasn't already causing issues! Domains are not queried or scoped in for most team objects

Reply:
I made a similar change in another branch